contents/data_vis_makie_latex.md

```diff
@@ -30,6 +30,11 @@ A more involved example will be one with some equation as `text` and increasing
 ```jl
 @sco JDS.multiple_lines()
 ```
+<!--
+The `lines` labels can be "simplified" to `L"%$i x"`. Then there also would be
```

Comment:
Even though LaTeXStrings if loaded with Makie I want the package loading to remain explicit. That way if the user wants can google it and go directly to that package if needed.

Reply:
That is perfectly fine for me. Nonetheless you should use the "simplified" version (everywhere) and optimally add a sentence why in LaTeXStrings variables (`$i`) need to be prefixed with `%`, i.e. `%$i`. (To distinguish it from the start of a mathematical expression.)